helper/schema/valuetype_string.go

```diff
@@ -9,7 +9,7 @@ const _ValueType_name = "TypeInvalidTypeBoolTypeIntTypeFloatTypeStringTypeListTy
 var _ValueType_index = [...]uint8{0, 11, 19, 26, 35, 45, 53, 60, 67, 77}
 
 func (i ValueType) String() string {
-	if i < 0 || i >= ValueType(len(_ValueType_index)-1) {
+	if i < 0 || i+1 >= ValueType(len(_ValueType_index)) {
```

Comment:
@svanharmelen were these this `stringer` changes? seems like this diff is unrelated to the test change... did it accidentally sneak in?